e2e/commands_test.go

```diff
@@ -178,86 +181,6 @@ func TestDetectApp(t *testing.T) {
-
-	tmpDir := fs.NewDir(t, t.Name())
-	defer tmpDir.Remove()
-
-	// Running a docker in docker to bundle the application
-	dind := NewContainer("docker:18.09-dind", 2375)
-	dind.Start(t)
-	defer dind.Stop(t)
-
-	// Create a build context
-	cmd.Command = dockerCli.Command("context", "create", "build-context", "--docker", fmt.Sprintf(`"host=tcp://%s"`, dind.GetAddress(t)))
-	icmd.RunCmd(cmd).Assert(t, icmd.Success)
-
-	// The dind doesn't have the cnab-app-base image so we save it in order to load it later
-	cmd.Command = dockerCli.Command("save", fmt.Sprintf("docker/cnab-app-base:%s", internal.Version), "--output", tmpDir.Join("cnab-app-base.tar.gz"))
-	icmd.RunCmd(cmd).Assert(t, icmd.Success)
-
-	cmd.Env = append(cmd.Env, "DOCKER_CONTEXT=build-context")
-	cmd.Command = dockerCli.Command("load", "-i", tmpDir.Join("cnab-app-base.tar.gz"))
-	icmd.RunCmd(cmd).Assert(t, icmd.Success)
-
-	testCases := []struct {
-		name           string
-		cmd            []string
-		invocImage     string
-		expectedBundle string
-	}{
-		{
-			name:           "simple-bundle",
-			cmd:            dockerCli.Command("app", "bundle", filepath.Join("testdata", "simple", "simple.dockerapp"), "--output", tmpDir.Join("simple-bundle.json")),
-			invocImage:     "simple:1.1.0-beta1-invoc",
-			expectedBundle: "simple-bundle.json.golden",
-		},
-		{
-			name:           "bundle-with-tag",
-			cmd:            dockerCli.Command("app", "bundle", filepath.Join("testdata", "simple", "simple.dockerapp"), "--tag", "myimage:mytag", "--output", tmpDir.Join("bundle-with-tag.json")),
-			invocImage:     "myimage:mytag-invoc",
-			expectedBundle: "bundle-with-tag.json.golden",
-		},
-	}
-	for _, tc := range testCases {
-		t.Run(tc.name, func(t *testing.T) {
-			testDir := fs.NewDir(t, "")
-			defer testDir.Remove()
-
-			// Bundle the docker application package to a CNAB bundle, using the build-context.
-			cmd.Command = tc.cmd
-			icmd.RunCmd(cmd).Assert(t, icmd.Success)
-
-			// Check the resulting CNAB bundle.json
-			golden.Assert(t, string(golden.Get(t, tmpDir.Join(tc.name+".json"))), tc.expectedBundle)
-
-			// List the images on the build context daemon and checks the invocation image is there
-			cmd.Command = dockerCli.Command("image", "ls", "--format", "{{.Repository}}:{{.Tag}}")
-			icmd.RunCmd(cmd).Assert(t, icmd.Expected{ExitCode: 0, Out: tc.invocImage})
-
-			// Copy all the files from the invocation image and check them
-			cmd.Command = dockerCli.Command("create", "--name", "invocation", tc.invocImage)
-			id := strings.TrimSpace(icmd.RunCmd(cmd).Assert(t, icmd.Success).Stdout())
-			cmd.Command = dockerCli.Command("cp", "invocation:/cnab/app/simple.dockerapp", testDir.Join("simple.dockerapp"))
-			icmd.RunCmd(cmd).Assert(t, icmd.Success)
-			cmd.Command = dockerCli.Command("rm", "--force", id)
-			icmd.RunCmd(cmd).Assert(t, icmd.Success)
-
-			appDir := filepath.Join("testdata", "simple", "simple.dockerapp")
-			manifest := fs.Expected(
-				t,
-				fs.WithMode(0755),
```

Comment:
I think part of this test is still valid, as it checks the built invocation image. We should still check that everything is OK after a build. WDYT?

Reply:
Right! This should be transferred as a unit test 👍 And integrated in a broader test suite around the built invocation image.